pkg/cli/builder/table/funcs.go

```diff
@@ -272,46 +272,58 @@ func DisplayRange(minVal, maxVal any) (string, error) {
 }
 
 func DefaultMemory(obj any) (string, error) {
-	b, ok := obj.(adminv1.ComputeClassMemory)
-	if !ok {
-		return "", fmt.Errorf("object passed is not a ComputeClassMemory struct")
+	switch b := obj.(type) {
+	case apiv1.ComputeClassMemory:
+		return defaultMemory(b.Default, b.Max), nil
+	case adminv1.ComputeClassMemory:
+		return defaultMemory(b.Default, b.Max), nil
+	default:
+		return "", fmt.Errorf("object passed cannot be converted into a ComputeClassMemory struct")
 	}
+}
 
-	result := b.Default
-	if b.Default == "0" || b.Default == "" {
-		result = b.Max
+func defaultMemory(def, max string) string {
```

Comment:
This function looks incorrect. `def` is always returned instead of `result`.

Reply:
Yep, you're right. Fixed.